### ISSUE TITLE
Link errors due to remove duplicates

### ISSUE THREAD

**Issue by mgates3** [icl-utk-edu/blaspp#64]
`list( REMOVE_DUPLICATES BLAS_LAPACK_LIBRARIES )` will remove duplicates, but the resulting list can be in the wrong order for linking. Here's a diff before and after this changed in BLAS++, where `blis` now comes only before `lapack` (edited for readability):
```
>   INTERFACE_LINK_LIBRARIES
    "/.../libblis.a;
    m;
    Threads::Threads;
    /.../liblapack.a;
    /.../libgfortran.so"
---
<   INTERFACE_LINK_LIBRARIES
    "/.../libblis.a;
    m;
    Threads::Threads;
    /.../liblapack.a;
    /.../libblis.a;
    m;
    Threads::Threads;
    /.../libgfortran.so"
```

**Comment by mgates3** [comment]
@ayarkhan Did adding `REMOVE_DUPLICATES` solve a problem with oneMKL, or was it just added to reduce clutter in the list of libraries?

**Comment by ayarkhan** [comment]
I removed `list( REMOVE_DUPLICATES BLAS_LAPACK_LIBRARIES )` from blaspp and lapackpp CMake.
And checked that this does not cause any differences on building on sunspot/Intel.
There are a couple of very small PRs for this.

**Comment by mgates3** [comment]
BLAS++ PR #67 and LAPACK PR https://github.com/icl-utk-edu/lapackpp/pull/46

**Comment by mgates3** [comment]
Fixed in BLAS++ PR #67 and LAPACK PR https://github.com/icl-utk-edu/lapackpp/pull/46
